dashboard/app/controllers/admin_search_controller.rb

```diff
@@ -73,6 +73,9 @@ def undelete_section
     redirect_to :lookup_section
   end
 
+  def pilots
```

Comment:
For my understanding...these blank methods are required if you want an action to have some sort of before_action happen?

Reply:
@davidsbailey I don't know the answer to this. Want to help Ben and I out?

Reply:
these aren't necessary, and there is no explicit guidance in the rails docs on whether to include them anyway. I would suggest searching the internet for opinions on whether to include empty rails controller methods :-) I don't feel strongly one way or the other here.